host_vars/pulsar-high-mem1/pulsar-high-mem1.yml

```diff
@@ -24,7 +24,7 @@ use_internal_ips: false
 
 # cvmfs
 cvmfs_cache_base: /mnt/var/lib/cvmfs
-cvmfs_quota_limit: 250000
+cvmfs_quota_limit: 819200
```

Comment:
Where does this 'magic' number come from?

Reply:
800 times 1024

Reply:
more than tripling in size from previously

Reply:
Yes. There is a 10TB disk that is rarely more than half full, and a lot of large databases in cvmfs that are being accessed for kraken2 jobs.  This seems like good use of the space.